customer-terminate-service/app/src/main/java/se/inera/intyg/cts/infrastructure/configuration/AppConfig.java

```diff
@@ -1,23 +1,93 @@
 package se.inera.intyg.cts.infrastructure.configuration;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import se.inera.intyg.cts.application.service.ExportService;
 import se.inera.intyg.cts.application.service.TerminationService;
+import se.inera.intyg.cts.domain.repository.CertificateBatchRepository;
+import se.inera.intyg.cts.domain.repository.CertificateRepository;
+import se.inera.intyg.cts.domain.repository.CertificateTextRepository;
+import se.inera.intyg.cts.domain.repository.TerminationRepository;
+import se.inera.intyg.cts.domain.service.CollectExportContent;
+import se.inera.intyg.cts.domain.service.CreatePackage;
+import se.inera.intyg.cts.domain.service.ExportPackage;
+import se.inera.intyg.cts.infrastructure.integration.GetCertificateBatch;
+import se.inera.intyg.cts.infrastructure.integration.GetCertificateTexts;
+import se.inera.intyg.cts.infrastructure.integration.IntegrationCertificateBatchRepository;
+import se.inera.intyg.cts.infrastructure.persistence.JpaCertificateRepository;
+import se.inera.intyg.cts.infrastructure.persistence.JpaCertificateTextRepository;
 import se.inera.intyg.cts.infrastructure.persistence.JpaTerminationRepository;
+import se.inera.intyg.cts.infrastructure.persistence.repository.CertificateEntityRepository;
+import se.inera.intyg.cts.infrastructure.persistence.repository.CertificateTextEntityRepository;
 import se.inera.intyg.cts.infrastructure.persistence.repository.TerminationEntityRepository;
+import se.inera.intyg.cts.infrastructure.service.CreateEncryptedZipPackage;
 
 @Configuration
 public class AppConfig {
 
```

Comment:
Was playing around and trying out explicit bean definitions instead of using stereotypes and component scanning. But it´s rather painful and I don't see a good reason why. Good that you raised the questions, because I don't have a good reason... :-) Have added Spring stereotypes to all beans (except domain package where we don't want to bring in Spring as a dependency) and rely on component scanning.